test/integrationtests/skills/message_tester.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+    This message tester lets a user input a Message event and a json test
+    case, and allows the evaluation of the Message event based on the
+    test case
+
+    It is supposed to be run in the Mycroft virtualenv, and python-tk
+    must be installed (on Ubuntu: apt-get install python-tk)
+"""
+
+from Tkinter import Label, Button, Tk, NORMAL, END, DISABLED
+import ScrolledText
+import skill_tester
+import ast
+
+EXAMPLE_EVENT = "{'expect_response': False, \
+                'utterance': u'Recording audio for 600 seconds'}"
+
+EXAMPLE_TEST_CASE = '{ \n\
```

Comment:
Why don't we change this to triple quotes since that will automatically keep newlines. Then it would look like:
```python
EXAMPLE_TEST_CASE = '''
{
  "utterance": "record",
  "intent_type": "AudioRecordSkillIntent", ...
'''
```

Reply:
Yes, that is much cleaner. I know @forslund has already made some changes, maybe for this small thing it might be easier for him to do it, to avoid merges?

Reply:
@CarstenAgerskov Should be totally fine. I messaged him and he said he hasn't made any permanent changes or anything.

Reply:
Changed as suggested